dora/structures/emails.py

```diff
@@ -128,9 +128,9 @@ def send_orphan_structure_notification(structure):
     context = {
         "recipient_email": structure.email,
         "structure_name": structure.name,
-        "doc_link": "https://aide.dora.inclusion.beta.gouv.fr/fr/article/decouvrir-et-faire-decouvrir-dora-1nyj6f1/",
+        "dora_doc_link": "https://aide.dora.inclusion.beta.gouv.fr/fr/article/decouvrir-et-faire-decouvrir-dora-1nyj6f1/",
         "webinar_link": "https://app.livestorm.co/dora-1/presentation-dora",
-        "cta_link": f"{settings.FRONTEND_URL}/structures/{structure.slug}?mtm_campaign=MailsTransactionnels&mtm_kwd=InvitationStructuresOrphelines",
+        "cta_link": f"{settings.FRONTEND_URL}/auth/rattachement?siret={structure.siret}&mtm_campaign=MailsTransactionnels&mtm_kwd=InvitationStructuresOrphelines",
```

Comment:
```suggestion
        "cta_link": f"{settings.FRONTEND_URL}/auth/invitation?structure={structure.slug}&mtm_campaign=MailsTransactionnels&mtm_kwd=InvitationStructuresOrphelines",
```

Il vaut mieux passer par ce lien, ça permet de prendre en compte les antennes (qui n'ont pas de siret), et la page d'arrivée est plus éloquente "Vous avez reçu une invitation pour rejoindre la structure suivante".  À retester quand même :sweat_smile: 

En gros la page rattachement est faite pour la navigation à l'intérieur du front, en particulier le lien interne "Adhérer à une autre structure", ou en redirection automatique pour les gens qui se connectent et qui ne sont rattachés à rien. La page "invitation" est spécifique... aux invitations; cela dit la différence entre les deux est faible, surtout depuis la semaine dernière ou j'ai du mettre en place le rattachement aux antennes France Travail, et où la page de rattachement accepte plus ou moins les slugs. Bref, on pourra sans doute les faire re-converger un peu à terme.

Reply:
ach, c'est Yoan qui m'avait demandé d'éventuellement déplacer ça.
Explication très claire, merci !

Reply:
ouais, on arrivait sur la page d'accueil de la structure avant c'était pas top; c'est clairement mieux d'arriver sur la page d'invitation.

Reply:
j'effectue les modifications en conséquence et je plie ça dans la journée